src/java.compiler/share/classes/javax/lang/model/element/TypeElement.java

```diff
@@ -88,7 +88,7 @@
 public interface TypeElement extends Element, Parameterizable, QualifiedNameable {
     /**
      * Returns the type defined by this class or interface element,
-     * returning the <i>prototypical</i> type for an element
+     * returning the <dfn>{@index "prototypical type"}</dfn> for an element
```

Comment:
Since we are here - another term the JLS uses for synthetic types that are given to some expression (for the purpose of some computation) is _notional type_ which might be easier on the eye.

Reply:
Hmm. Okay, from searching through the JLS 20 pdf "notional type" looks to be introduced in 4.9 "Intersection Types". The term looks to be introduced in the JLS for Java SE 8.

JSR 269 has used "prototypical type" since its introduction, so for this changeset I plan to keep that terminology.

As future work, it could be helpful to take a pass over the javax.lang.model.* type discussion to see how it could better align with evolution of JLS terminology since the API was introduced.

A general discussion of denotable and non-denotable types might be helpful too.